src/components/Input/Input.tsx

```diff
@@ -90,11 +94,27 @@ function Input(props: Props) {
           {endIcon && (
             <div
               onClick={endIconClick}
-              className="absolute right-3 top-1/2 -translate-y-1/2 cursor-pointer text-panel-text"
+              className={cx(
+                'absolute right-3 top-1/2 -translate-y-1/2 cursor-pointer text-panel-text',
+                endIconClassName,
+              )}
             >
               <Icon path={endIcon} size={1} horizontal vertical rotate={180} />
             </div>
           )}
+          {endIcons?.length && (
```

Comment:
`endIcon`,`endIconClassName` and `endIconClick` seem redundant now.
Places which are using `endIcon` can be refactored to use `endIcons` instead